### PR TITLE
Update ringct.md

### PR DESCRIPTION
**Pull request by SamsungGalaxyPlayer**
I tried to simplify the wording slightly. Includes a few grammar fixes. I also think we need a conclusion, so I added the heading for one.

### CODE REVIEW

File: ringct.md [hunk 1]
```diff
@@ -17,51 +17,50 @@ This is accomplished by the digital signature, which can only be proven to be si
...
 With this input, you cryptographically commit to some amount without revealing it.
 With RCT, instead of revealing the values 12.56, 10.06, and 2.5, you cryptographically commit to them.
 The commitment will look like random numbers, but they actually have important features which ensures safety of the mechanism.
-To verify that no new money is created, the sum of inputs minus the sum of outputs must be 0.
+To verify that no new money is created, the sum of inputs must equal the sum of the outputs.
 This is easy to check if you see the numbers.
 With commitments, it can be checked exactly the same way!
-The only difference is that you're summing some seemingly random numbers, but the sum must again be 0.
-The underlying cryptography guarantees that if the sum of commintment is 0, then also the sum of underlying masked values must be 0.
+The only difference is that you're comparing seemingly random numbers, but they must still will be the same on both sides of the transaction.
```
Comment (JollyMort) [inline ringct.md]: Can we reword to: The only difference is that you're comparing seemingly random numbers, but they can still be summed to check both sides of the transaction.

File: ringct.md [hunk 2]
```diff
@@ -17,51 +17,50 @@ This is accomplished by the digital signature, which can only be proven to be si
...
+The underlying cryptography guarantees that if both numbers are the same, then the amount being sent is the same as the amount of funds available.
```
Comment (JollyMort) [inline ringct.md]: and this one: The underlying cryptography guarantees that a check performed on commitments means that the amount being sent is the same as the amount of funds available.

**Review by JollyMort — CHANGES_REQUESTED** [review]

**Comment by SamsungGalaxyPlayer** [comment]
@JollyMort made your changes
